TaskfMRIAnalysis/scripts/TaskfMRILevel1.sh

```diff
@@ -190,6 +190,46 @@ log_Msg "MAIN: SET_NAME_STRINGS: RegString: ${RegString}"
 log_Msg "MAIN: SET_NAME_STRINGS: ParcellationString: ${ParcellationString}"
 log_Msg "MAIN: SET_NAME_STRINGS: Extension: ${Extension}"
 
+##### CHECK_FILES: Check that necessary inputs exist before trying to use them #####
+# Assemble list of input filenames that need to be checked
+Filenames="";
```

Comment:
This could be done with whitespace-safety like so:

```
Filenames=()

if <whatever>; then
    Filenames+=("$somefile")
fi
...
missingfiles=()
for name in "${Filenames[@]}"; do
    if ! [[ -e "$name" ]]; then
        missingfiles+=("$name")
    fi
done
if (( ${#missingfiles[@]} > 0 )); then
    log_Err_Abort "Missing necessary input files: ${missingfiles[*]}"
fi
```